tests/test_all_rules.py

```diff
@@ -232,7 +232,7 @@ def test_required_tags(self):
             'endgame-*': {'all': ['Endpoint']},
             'logs-aws*': {'all': ['AWS']},
             'logs-endpoint.alerts-*': {'all': ['Endpoint']},
-            'logs-endpoint.events.*': {'any': ['Windows', 'macOS', 'Linux']},
+            'logs-endpoint.events.*': {'any': ['Windows', 'macOS', 'Linux', 'Endpoint']},
```

Comment:
```suggestion
            'logs-endpoint.events.*': {'any': ['Windows', 'macOS', 'Linux', 'Host']},
```

Use `Host` for the rule tags instead